reviews/2020-Network Service Mesh-annual.md

```diff
@@ -0,0 +1,192 @@
+that Network Service is available to their Pods (not to the cluster as a whole) without using CNI.  Standard 
+Kubernetes Networking is still provided by CNI.  Network Service Mesh takes care to insure that it is completely 
+non-conflicting with Kubernetes Networking.
+
+### Example: **db-replication**
+
+Database workloads across multiple K8s clusters in different public clouds and on prem can 
+connect to a vL3 Network Service that exists solely for the purpose of database replication, and only has
+database workloads that need to participate in such replication connected to it.  This is done declaratively.  The developer
+simply adds a simple one line annotation to the Pod spec for their DB Pods. 
+
+## Alignment with Cloud Native
+
+### Loose Coupling
+IP Networking has traditionally been strongly coupled to the runtime domain (K8s cluster, VIM, datacenter, etc).
+Where a workload ran determined what networking was available to it and the semantics of what networking features
+it could receive.
+
+Network Service Mesh allows workloads to be 'loosely coupled' to the Network Service (or Network Services) that are
+relevant to their particular work independent of the environment they are running in.  Workloads running in different
+clusters hybrid/multi-cloud scenarios can connect to a common Network Service that provides the networking features
+needed for their particular collaboration.
+
+Network Service Mesh does this without disturbing (and taking care not to break) the existing networking provided
+by the runtime.  K8s Networking via CNI is untouched, and continues to work as expected.
+
+Network Service Mesh makes IP networking itself Cloud Native by loosely coupling with the existing 
+immutable infrastrcture in K8s to deliver whatever non-standard Networking needs a workload has with minimal toil.
+
+NSM is a complement to higher level L7 Service Meshes (Linkerd), providing an additional 
+connectivity,security and observability at lower layers.
+
+### Minimal Toil
+The Network Service Mesh enabled developers access Network Services with minimal toil:
+
+* Developers ask for what Network Service they want by name
+* Developers can add metadata to their request for a Network Service with labels.
+* Developers simply get the Network Service they asked for
+
+Developers never have to know or think about 35 year old legacy Networking concepts like:
+
+* Subnets
+* Interfaces
+* Routes
+* IPAM
+
+### Immutable Infrastructure
+
+Network Service Mesh shifts IP networking from being 'Infrastructure' to being a selection of 'Network Services'.
+This allows infrastructure to remain immutable while meeting a much wider variety of requirements.  
+By scoping the selection of 'Network Services' to the granularity of workloads (rather than clusters), Network Service Mesh
+allows different workloads to consume different Network Services that could be conflicting if forced into cluster level
+infrastructure.
+
+## Highlights of Last Year
```

Comment:
Can we include information about the new features developed/shipped during the last year?

Reply:
Absolutely, we'll do that soon.